src/transform/roads.py

```diff
@@ -0,0 +1,45 @@
+import sys
+
+import geopandas as gpd
+import fiona
+from jsonschema import validate
+
+from utils.yaml_api import parse_yaml
+
+GADM_FILENAME = 'gadm36_{ISO3}.gpkg'
+GADM_LAYER = 'gadm36_{ISO3}_0'
+
+
+def transform_cod():
+    transform('cod', sys.argv[1], sys.argv[2], sys.argv[3])
+
+
+def transform_gadm():
+    transform('gadm', sys.argv[1], sys.argv[2], sys.argv[3])
+
+
+def transform(source: str, input_filename: str, schema_filename: str, output_filename: str):
+    """
+    :param source: "cod" or "gadm"
+    """
+    config = parse_yaml('config.yml')
+
+    if source == "cod":
+        df_roads = gpd.read_file(f'zip://{input_filename}')
+
+#TODO Fix schema check, CRS, column names and validation
+#        schema_mapping = {
+#            'admin0Name_en': 'name_en'
+#        }
+
+    # Change CRS
+#   df_roads = df_roads.to_crs(config['constants']['crs'])
```

Comment:
For some reason the COD file contains roads with empty geometries. If you add this line above:
`df_roads = df_roads[df_roads['geometry'].notna()]`
then the CRS change will work.